extensions/amp-social-share/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {SocialShare} from '../social-share';
+import {object, select, text, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+
+export default {
+  title: 'Social Share',
+  component: SocialShare,
+  decorators: [withA11y, withKnobs],
+};
+
+export const _default = () => {
+  const knobConfigurations = [
+    'email',
+    'facebook',
+    'linkedin',
+    'pinterest',
+    'tumblr',
+    'twitter',
+    'whatsapp',
+    'line',
+    'sms',
+    'system',
+    'custom endpoint',
+    undefined,
+    '',
+    'random',
+  ];
+  const type = select('type', knobConfigurations, knobConfigurations[0]);
+  const href = text('shareEndpoint', 'Not Specified');
+  const params = object('params', {'subject': 'test'});
+  const bindings = object('bindings', {
```

Comment:
Again, ok to defer to a follow up PR, but this prop bugs me the most. Putting on a user's hat on, I really don't understand why there are two `params` and `bindings`, and what's the difference between them.

Reply:
Sounds good, I have created a comment in the parent issue to track outstanding items.

https://github.com/ampproject/amphtml/issues/28283#issuecomment-628231428